source/plugins/ruby/in_kubestate_hpa.rb

```diff
@@ -7,6 +7,7 @@ module Fluent::Plugin
   class Kube_Kubestate_HPA_Input < Input
     Fluent::Plugin.register_input("kubestate_hpa", self)
     @@istestvar = ENV["ISTEST"]
+    @@totalHPACount = 0
```

Comment:
same here.  we can have instance variable @totalHPACount instead of class variable @@totalHPACount.